### ISSUE TITLE
S3 as file storage

### ISSUE THREAD

**Issue by Wouter0100** [tiredofit/docker-freescout#63]
**Description of the feature**
The ability to use an S3 bucket as file storage. This is possible by providing custom ENV vars.

**Benefits of feature**
Being able to move around easier and not relying on local storage for backups. S3 has in general a better data guarantee.

**Comment by tiredofit** [comment]
I would appreciate that too, but it would be best to make this feature suggestion in the actual repository not just the Docker image as they are two seperate things. Note, I have it working under S3 but it needs manual work on the users end to setup via S3QL.

**Comment by Wouter0100** [comment]
Yeah, I know - but there was an issue in the Freescout repository that was closed because it is already possible by correctly configuring Laravel. That's why I opened it here, as it would be awesome to have that as part of the docker image.

**Comment by Nimdy** [comment]
You could deploy into fargate and use EFS for mapping of persistent storage... This is something I do already.
